cubedash/summary/_summarise.py

```diff
@@ -29,7 +29,19 @@
 
 _NEWER_SQLALCHEMY = not sqlalchemy.__version__.startswith("1.3")
 
-DEFAULT_TIMEZONE = environ.get("CUBEDASH_DEFAULT_TIMEZONE", "Australia/Darwin")
+# Get default timezone via CUBEDASH_SETTINGS specified config file if it exists,
+# otherwise default to Australia/Darwin
+default_timezone = "Australia/Darwin"
+settings_file = os.environ.get("CUBEDASH_SETTINGS", "settings.env.py")
```

Comment:
Ewww, this isn't good at all. We'll have to do something better. But if it works, it can be in a later PR.

Reply:
The only alternative I could think of was to manually create a Flask Config object and then use its `from_pyfile` method, but neither solution seemed good so I went with this one to avoid importing flask here for this sole purpose. It should hopefully work, but I think the default timezone logic is in dire need of a larger rework anyway.
If there's a less painful interim solution, I'd be happy to implement it.

Reply:
This is fine for now, it's ugly, but I like that it's contained in a single file that we can clean up in future.